src/app/forms/components/read-only/read-only.component.html

```diff
@@ -1 +1,7 @@
-{{ value | defaultNullOrEmpty }}
+<div>
+  <h1 *ngIf="label" class="govuk-label-wrapper">
+    <label class="govuk-label govuk-label--m" for="{{ name }}">{{ label }}</label>
+  </h1>
+
+  <span class="govuk-body govuk-body-m" id="{{ name }}">{{ value }}</span>
```

Comment:
what does this class do? I thought the point of this component was to display the plain text value of the control.

Reply:
`govuk-body` adds the necessary styling in terms of font type and weight. `govuk-body-m` has been omitted.